README.md

```diff
@@ -1,3 +1,101 @@
+
+## Introduction
+
+An app built as proof of portability for [Knative](https://knative.dev) that provides a platform for event organizers to share info during large events like concerts, parades, expos, etc. Key features of this app include:
+* Allow authorized event organizers to create and edit events
+* Handle uploading and serving multimedia posts made by attendees
+* Organize and filter multiple sub-events
+* Provide a clean front-end web UI for interaction with the service
+* Authenticate with OAuth 2.0
+
+This app is built as part of a proof of portability project for [Knative](https://knative.dev). It is meant to show the key features of Knative, to test the conformance various cloud product implementations of Knative like [Google Cloud Run](https://cloud.google.com/run/), and to document with functioning sample code how one might develop, build, and deploy with Knative. Key features of Knative this app demonstrates include:
+* Develop microservices independently that can be individually deployed, updated, and auto-scaled
+* Deploy containerized code with minimal configuration, allowing the developer to focus on features rather than infrastructure
+* Run serverless, stateful containers that enable pay-for-use billing
+
+### A note on microservices
+This project is organized into microservices. Although this repository contains a few top level files such as this README.md, each direct subfolder of this project is its own independent microservice. This means a few things: 
+* Each microservice subfolder is meant to be built and deployed independently from each other
+* The microservices interact with each other in deployment through HTTP requests
+    * You will need to supply environmental variable to the microservices specifying each others' endpoints
+* Each subfolder has its own README.md with more specific information regarding building, testing, deployment, etc.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system. Because of this microservice architecture, each subfolder will need to be deployed as its own independent subfolder. For specific instructions, see the README.md file in each subfolder, but generally, they all follow the following pattern:
+
+### Prerequisites
+
+These microservices are written in [Python 3](https://www.python.org/) and should run on Python 3.6+.
+
+### Installing
+
+How to install and run each microservice for local development.
+
+Make sure your working directory is the microservice's subfolder. For example, for the "users" microservice:
+
+```
+cd users
+```
+
+Set up and activate a virtual environment.
+
+```
+python3 -m venv venv && . venv/bin/activate
+```
+
+Install the required python modules.
+
+```
+pip3 install requirements.txt
```

Comment:
install **-r** requirements.txt ?

Reply:
Done https://github.com/knative-portability/large-events/pull/6/commits/d43b1a3a7ca0eb76e8ab5f58c3578202af25cbd4

Reply:
Thanks for the catch